src/windows/wintab.c

```diff
@@ -272,19 +274,27 @@ void wintab_on_packetext(struct wintab *ctx, MTY_Event *evt, const PACKETEXT *pk
 	const SLIDERDATA *curr_ring = &pktext->pkTouchRing;
 	const SLIDERDATA *prev_ring = &ctx->prev_pktext.pkTouchRing;
 	if (curr_ring->nPosition != prev_ring->nPosition || curr_ring->nMode != prev_ring->nMode) {
+		uint16_t position = (uint16_t) curr_ring->nPosition;
+		if (position >= 73) // If outside of the logical 0-72 range, offset it back for event consistency
+			position -= UINT8_MAX - 73;
+
```

Comment:
`curr_ring->nPosition` is 32 bits, so the value could be 65535,
Thus position is larger than 73, then position -= 255 - 73, which means that the position is still larger than 73. Do we just need to set position to 73? do we need to see if the position is larger than 255?

Reply:
We ended up with this after extensive testing of the extension, where we've seen this behavior:
* `nPosition` will range from 0 to 72 here, and can't be greater than that in a normal scenario (I think Jason can confirm)
* In case a custom settings for `parsecd` are configured in the Wacom settings, the position is "inverted" within a byte range, so it ranges from 182 to 255 (no clue why, and this is not documented anywhere)

An option to make this check safer would be to set the value to zero if the position is out of the known ranges. That way, the driver's behavior is known. Would that fit?